voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
@gernest yeah, i think @nshi couldn't get why we use code block instead of url.Port(). 
So i've tried to explain why we use it. 
Besides of it, i think, using code block won't be a problem if we have/give 'licence'. 
Waiting for your thoughts :)